src/components/ModalPayment/SubmissionButton.tsx

```diff
@@ -7,18 +7,22 @@ type Props = {
 
 const SubmissionButton = ({ canSubmit }: Props) => {
   const context = useContext(Context);
+  var [submittable, setSubmittable] = useState(false);
 
   const handleSubmit = (evt: { preventDefault: () => void }) => {
     evt.preventDefault();
     context.onCreateNonce();
+    setSubmittable(false);
```

Comment:
When do we set this to true?

Reply:
Clicking on any of the inputs in the modal will re-render button and the button will be enabled again (assuming the criteria around name, email, and terms/conditions are met)

Reply:
Scratch that, it doesn't work. Gonna rework this PR

Reply:
Got it to work with some help from @ath-chen ! Check out the updated video https://drive.google.com/open?id=1m51ZlR56oLG4DVIsVw8YMa9r6LXxVKrJ